tidy3d/components/simulation.py

```diff
@@ -4322,6 +4323,8 @@ def subsection(
             Remove structures outside of the new simulation domain.
         remove_outside_custom_mediums : bool = True
             Remove custom medium data outside of the new simulation domain.
+        include_pml_cells : bool = False
+            Keep PML cells in simulation boundaries.
```

Comment:
we can add something like "Note that retained PML cells will be converted to regular cells, and the simulation domain boundary will be moved accordingly."